### PR TITLE
Output messages now written through python loggers

### LINKED ISSUE #30: Add logging
Especially once we get to distributed runs, it will be a lot easier to track down problems if we have a robust logging system.

### LINKED ISSUE #44: Make logging directory configurable
Allow users to set the logging directory in the `general` section of the configuration file.

### PR DESCRIPTION
**Pull request by rplzzz**
Fixes #30 
Fixes #44

### CODE REVIEW

**Comment by rplzzz** [comment]
Also, note that this was branched from #37, so it's probably best to deal with that one first, before tackling this one.

**Comment by rplzzz** [comment]
Anything else we need on this one?

File: cassandra/mp.py [hunk 1]
```diff
@@ -52,21 +53,21 @@ def bootstrap_mp(argvals):
 
     if argvals.logdir is None:
         logdir = 'logs'
+        os.makedirs(logdir, exist_ok=True)
```
Comment (rplzzz) [inline cassandra/mp.py]: I think we want to do this irrespective of whether we are using the default log location.

Comment (calebbraun) [reply cassandra/mp.py]: Good point, I'll update it.

**Pull request merged**